run_cv.py

```diff
@@ -3,13 +3,15 @@
 import argparse
 import cv2
 from pose_calculator import PoseCalculator
-# from depth_data_generator import DepthDataGenerator
-# from depth_live_generator import DepthLiveGenerator
+from depth_data_generator import DepthDataGenerator
+from depth_live_generator import DepthLiveGenerator
```

Comment:
Minor little nitpick, but can we move the DepthLiveGenerator import to after we check which generator was selected (currently line 36)? This way if someone doesn't have pyrealsense installed they can still run this file.